deno/rest/v10/channel.ts

```diff
@@ -594,7 +594,12 @@ export type RESTPostAPIGuildForumThreadsJSONBody = RESTPostAPIChannelMessagesThr
 /**
  * https://discord.com/developers/docs/resources/channel#start-thread-in-forum-channel
  */
-export type RESTPostAPIGuildForumThreadsFormDataBody = RESTPostAPIChannelMessageFormDataBody;
+export type RESTPostAPIGuildForumThreadsFormDataBody = {
+	/**
+	 * First message in the forum thread
+	 */
+	message: RESTPostAPIChannelMessageFormDataBody;
+} & RESTPostAPIChannelMessagesThreadsJSONBody;
```

Comment:
1. Invert the `&` (so start with the `RESTPostAPIChannelMessagesThreadsJSONBody`, and then & it with the message
2. The message should have the same type as `payload_json` from MessageFormData (ala string)